web/crux-ui/src/websockets/websocket-client.ts

```diff
@@ -6,6 +6,8 @@ import WebSocketClientEndpoint from './websocket-client-endpoint'
 import WebSocketClientRoute from './websocket-client-route'
 
 class WebSocketClient {
+  public static ERROR_SESSION_EXPIRED = 4000
```

Comment:
Please write a comment explaining why we need to use the range from 4000-4999 for the error codes. Also  stick with http based error codes like 4401 in this case.
So something like: `static ERROR_UNAUTHORIZED = 4401`